uni/lib/model/entities/exam.dart

```diff
@@ -58,6 +58,8 @@ class Exam {
     'Exames ao abrigo de estatutos especiais': 'EAE'
   };
 
+  static List<String> displayedTypes = types.keys.toList().sublist(0, 4);
+
```

Comment:
Try doing this in another way, the keys could return in another order if there is a new key, it's unordered.